src/main.go

```diff
@@ -0,0 +1,240 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/abcxyz/pkg/cli"
+)
+
+const (
+	githubContextEnv = "GITHUB_CONTEXT"
+	jobContextEnv    = "JOB_CONTEXT"
+)
+
+var rootCmd = func() cli.Command {
+	return &cli.RootCommand{
+		Name: "send-google-chat-webhook",
+		Commands: map[string]cli.CommandFactory{
+			"chat": func() cli.Command {
+				return &cli.RootCommand{
+					Name:        "workflownotification",
+					Description: "notification for workflow",
+					Commands: map[string]cli.CommandFactory{
+						"workflownotification": func() cli.Command {
+							return &WorkflowNotificationCommand{}
+						},
+					},
+				}
+			},
+		},
+	}
+}
+
+type WorkflowNotificationCommand struct {
+	cli.BaseCommand
+	flagWebhookUrl string
+}
+
+func (c *WorkflowNotificationCommand) Desc() string {
+	return "Send a message to a Google Chat space"
+}
+
+func (c *WorkflowNotificationCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options]
+
+  The chat command sends messages to Google Chat spaces.
+`
+}
+
+func (c *WorkflowNotificationCommand) Flags() *cli.FlagSet {
+	set := cli.NewFlagSet()
+
+	f := set.NewSection("Chat space options")
```

Comment:
By convention, section names should be all capital letters: https://github.com/abcxyz/readability/wiki/Go-Best-Practices#flag-sections